.circleci/config.yml

```diff
@@ -147,7 +147,7 @@ commands:
               export RELEASE_TAG='-t release'
             fi
             mkdir -p logs
-            make html O="-T $RELEASE_TAG -j4 -w /tmp/sphinxerrorswarnings.log"
+            make html O="-T $RELEASE_TAG -j1 -w /tmp/sphinxerrorswarnings.log"
```

Comment:
This change is because `pydata-sphinx-theme` doesn't support parallel writes: https://github.com/pydata/pydata-sphinx-theme/issues/1643. It doesn't seem to impact build time significantly though, [18m19s before](https://app.circleci.com/pipelines/github/matplotlib/matplotlib/30134/workflows/05155263-4a0d-4878-ac00-24baded525f4/jobs/82101) and [18m57s after](https://app.circleci.com/pipelines/github/matplotlib/matplotlib/30138/workflows/ece60c42-b04a-4b8b-93dc-27cb7959c8c7/jobs/82105).

Reply:
Not tested with the new version, but IIRC, I got significant speedups on my local machine with -j4. If we can't have that anymore, we need to update https://matplotlib.org/devdocs/devel/document.html#build-the-docs. Also, > 10min doc build times makes working on docs very unattractive. As [somebody else said](https://github.com/pydata/pydata-sphinx-theme/issues/1643#issuecomment-1922437653)

>  In practice, that goes from "let me quickly build the docs to see the effect of this change" to "let's just hope it renders correctly".

Are the other improvements so significant that we are willing to take this on?

Reply:
The comparison is on circleCI between 591cd650b39f8bbfb445e10f4b72ecec08309e62 and f178f5c52ab5a518cd9655633d876442a5f4b9df, the only difference being changing `-j4` to `-j1`. Perhaps even with `-j4` circleCI wasn't using any parallelism to build the docs?

> Are the other improvements so significant that we are willing to take this on?

They include at least some accesibility improvements which I would advocate for.

Reply:
Note that this is changing the parallel option only for the circleCI build, not any local or other builds, so the comparison using only times measured on circleCI is fair.

Reply:
And you can still use parallel options, it will just raise a warning at the end of the doc build that `pydata-sphinx-theme` isn't parallel write safe.

Reply:
I don't see any need to change our call here. The theme marks it self as parallel-unsafe, so requesting parallel build will be turned off anyway. And then when they fix themselves, we'd need to remember to turn it back on again.

Reply:
You could try to add a warnings filter to conf.py

Reply:
Do you mean the [suppress_warnings configuration](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-suppress_warnings)? It seems to be the only relevant option to do this, but warning about a non-parallel write safe theme doesn't seem to be convered by the list of warning types that can be suppressed.